src/react-apps/applications/ux-editor/src/actions/formDesignerActions/actions/createRepeatingGroup/index.ts

```diff
@@ -0,0 +1,33 @@
+import { Action } from 'redux';
+import * as ActionTypes from '../../formDesignerActionTypes';
+
+export interface ICreateRepeatingGroupAction extends Action {
+  id: string;
+}
+
+export interface ICreateRepeatingGroupFulfilled extends Action {
+}
+
+export interface ICreateRepeatingGroupRejected extends Action {
+  error: Error;
+}
+
+export function createRepeatingGroupAction(id: string): ICreateRepeatingGroupAction {
+  return {
+    type: ActionTypes.CREATE_REPEATING_GROUP,
+    id,
+  };
+}
+
+export function createRepeatingGroupFulfilled(): ICreateRepeatingGroupFulfilled {
```

Comment:
Ref comment about not extending Action interface. If you're removing the interface, just use this:
`export function createRepeatingGroupFulfilled(): Action`